controller-client/pom.xml

```diff
@@ -36,12 +36,7 @@
 
     <name>WildFly: Controller Client</name>
 
-    <properties>
-        <!-- maven-compiler-plugin -->
-        <!-- Keep the management client libs on JDK 7 -->
-        <maven.compiler.target>1.7</maven.compiler.target>
-        <maven.compiler.source>1.7</maven.compiler.source>
-    </properties>
+    <properties/>
```

Comment:
WildFly Elytron and Remoting 5 are now both Java 8 only so any clients using these will also be Java 8 as a minimum.